packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.tsx

```diff
@@ -69,6 +71,22 @@ function BlogPostItem(props: Props): JSX.Element {
     frontMatter.author_image_url ||
     frontMatter.authorImageURL;
 
+  // details on structured data support: https://developers.google.com/search/docs/data-types/article#non-amp
+  // and https://schema.org/BlogPosting
+  const blogPostStructuredData = {
+    '@context': 'https://schema.org',
+    '@type': 'BlogPosting',
+    headline: title,
+    description,
+    image: [image],
+    datePublished: date,
+    author: {
+      '@type': 'Person',
+      name: authorTitle,
+      url: authorImageURL,
```

Comment:
Shouldn't we use user's website rather than its URL avatar?

```suggestion
      url: authorURL,
```